src/Accounts/Authentication/Authentication.csproj

```diff
@@ -30,4 +30,8 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Can we remove this part? it seems it is not required.

Reply:
removed